mscore/inspector/inspector_accidental.ui

```diff
@@ -84,7 +84,7 @@
         <string>Reset value</string>
        </property>
        <property name="accessibleName">
-        <string>Reset Small value</string>
+        <string>Reset 'Small' value</string>
```

Comment:
Really, simple quote? I understand what you are trying to do but I really wonder if this is the appropriate way.